okio-files/src/jvmMain/kotlin/okio/Jvm.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2020 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okio
+
+import java.io.File
+
+val Path.file: File
```

Comment:
nit: `internal`?

Reply:
Fixed. Might promote this to public later.

Reply:
It would need to be `fun Path.toFile()` if it were public because this isn't really a property of a `Path`.